components/SponsorsSection.js

```diff
@@ -93,6 +93,13 @@ function ItemSponsor ({ name, src, type, url }) {
             padding: 0;
           }
         }
+        .sponsor-item-gold > a, .sponsor-item-silver > a{
```

Comment:
You can just use `a` here because `styled-jsx` is scoped to the parent element. It will not affect other links on this page. Isn’t that cool? 😁